From 89403a36d81df5681e9ffadac72231471f102a70 Mon Sep 17 00:00:00 2001
From: CI Assistant <yossi.ilkanaev@intel.com>
Date: Wed, 3 Apr 2019 14:45:44 +0200
Subject: [PATCH] Merge pull request #634 in SW_UGW/linux from
 bugfix/UGW_SW-32392-grx500-jumbo-support-fix-8.3.1 to 8.3.1

* commit 'fc39273cdad725fd012e1bf04ea1ec74b6450527':
  UGW_SW-32392 : Tuned jumbo frame length threshold with cbm driver's alignment, applicable for grx500 only.
---
 drivers/net/ethernet/lantiq/ltq_eth_drv_xrx500.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/net/ethernet/lantiq/ltq_eth_drv_xrx500.c b/drivers/net/ethernet/lantiq/ltq_eth_drv_xrx500.c
index 26eec9d9fc3f..4cd241282eb4 100644
--- a/drivers/net/ethernet/lantiq/ltq_eth_drv_xrx500.c
+++ b/drivers/net/ethernet/lantiq/ltq_eth_drv_xrx500.c
@@ -696,7 +696,7 @@ static int ltq_enable_gsw_r_pmac_jumbo(struct net_device *dev)
 	gsw_reg_set_bit(ops, 0xd05, 0x8);
 
 	/* Set the jumbo frame length threshold */
-	gsw_reg_set_val(ops, 0xd06, 1694);
+	gsw_reg_set_val(ops, 0xd06, 1662);
 
 	return 0;
 }
